config/locales/en.yml

```diff
@@ -5,6 +5,7 @@ en:
   date:
     formats:
       default: "%-d %b %Y"
+      iati: "%Y-%m-%d"
```

Comment:
It would be nice to mention this in the commit, although I can see the format here, making supet clear why there is a date format might help future us? Only needs to be: 

> Note that dates need to be in xs:Date format (YYYY-mm-dd) so we have added a new `iati` date format to the locales file

Or something like that?